tests/integration_tests/dashboards/security/security_dataset_tests.py

```diff
@@ -234,3 +235,4 @@ def test_get_dashboards_api_no_data_access(self):
         self.assert200(rv)
         data = json.loads(rv.data.decode("utf-8"))
         self.assertEqual(0, data["count"])
+        DashboardDAO.delete(dashboard)
```

Comment:
Previously this was non-idempotent.

Reply:
nice 👍 One more down, I wonder how many to go..